sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -69,6 +70,7 @@ export class Sender {
    *
    * @param message - Message to send.
    * @returns Promise<void>
+   * @throws {Error} Thrown if the underlying connection, client or sender is closed.
```

Comment:
`send()` and `sendBatch()` can throw `MessagingError` as well. Any reason not to document them?

Reply:
Updated error docs on all sender operations. Might have got missed out during commits